var/spack/repos/spack_repo/builtin/packages/vecgeom/package.py

```diff
@@ -7,16 +7,15 @@
 from spack_repo.builtin.build_systems.cuda import CudaPackage
 
 from spack.package import *
-from spack.variant import ConditionalVariantValues
 
 
 def _std_when(values):
```

Comment:
why is this different from the other _std_when?

Reply:
Cause in one case they only have conditional variants, in the other they have both conditional variants and strings